app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,58 @@
+class CommentsController < ApplicationController
+  before_action :set_comment, only: %i[ show edit update destroy ]
+
+  def index
+    @comments = Comment.all
+  end
+
+  def show
+  end
+
+  def new
+    @comment = Comment.new
+  end
+
+  def edit
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.build(comment_params)
+    @comment.user_id=current_user.id
+
+    respond_to do |format|
+      if @comment.save
+        format.html { redirect_to post_url(@post), notice: "Comment was successfully created." }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def update
+    respond_to do |format|
+      if @comment.update(comment_params)
+        format.html { redirect_to comment_url(@comment), notice: "Comment was successfully updated." }
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def destroy
+    @comment.destroy
+
+    respond_to do |format|
+      format.html { redirect_to comments_url, notice: "Comment was successfully destroyed." }
+    end
+  end
+
+  private
+    def set_comment
+      @comment = Comment.find(params[:id])
+    end
+
+    def comment_params
+      params.require(:comment).permit(:content)
+    end
+end
```

Comment:
add empty line at the end of file

Reply:
Done.

Reply:
add it end of the file (add after line 44 
)